tljh/config.py

```diff
@@ -172,6 +173,14 @@ def remove_config_value(config_path, key_path, value):
     with open(config_path, 'w') as f:
         yaml.dump(config, f)
 
+def check_hub_ready():
+    import requests
+
+    try:
+        r = requests.get('http://127.0.0.1:80')
```

Comment:
to avoid failures when https doesn't work:

```suggestion
        r = requests.get('http://127.0.0.1:80', verify=False)
```